test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,36 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Service Binding Request with Custom Environment Variables
```

Comment:
The indentation of this test is same as the previous test. See:

![Screenshot from 2020-09-07 15-13-20](https://user-images.githubusercontent.com/6303309/92374127-f335f780-f11c-11ea-97c5-3847e8fd84a0.png)

Reply:
I'm sorry for this I guess it's my mistake. Mea Culpa. But check for other indentation comments.

Reply:
Scenario description should reflect impact of a given user input (context) to the system. `Service Binding Request with Custom Environment Variables` does not tell us that at all.

My understand is that this PR is followup of https://github.com/redhat-developer/service-binding-operator/pull/602 where we [documented](https://github.com/redhat-developer/service-binding-operator/tree/master/examples/java_postgresql_customvar) how to refer in cusotm env variables sequences, scalars and map values from service CR.

The current scenario does perform the checks stated in [the example](https://github.com/redhat-developer/service-binding-operator/tree/master/examples/java_postgresql_customvar), but it should. The introduced custom env vars do not have any impact on the app - the scenario verifies the same flow as [already existing scenario](https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToService.feature#L11).

Instead, the scenario (or perhaps it should be splitted into multiple ones, for each source value type) should verify:

* in case when custom env var refer a sequence (e.g. TAGS here), then a number of env vars will be injected, following TAGS_X pattern, where X is the sequence number
* when custom env var refers a scalar value, only single env var name will be injected containing that value
* when a map is reffered, then again custom env var name serves as prefix, and a number of env vars gets whose names follow given pattern

The existence of each expected variable and its value should be checked. Without that, we cannot be sure if the code does what it expect.